src/pages/timeline.tsx

```diff
@@ -203,13 +184,30 @@ const Timeline: NextPage<TimelineProps> = ({ countries, ne, sw }) => {
     };
   }, [placeMarkers]);
 
+  const focusMarkersInCountry = useCallback(
+    (country: string) => {
+      const countryMarkers = markers.filter((marker) => marker.country === country);
+      const { sw: s, ne: n } = boundingCoordinates(countryMarkers);
+
+      map.current.fitBounds([s, n]);
+    },
+    [markers]
+  );
+
   return (
     <>
       <div className={styles.map} id="map" ref={mapContainer} />
 
       <div className={styles.list} ref={listContainer}>
         {countries.map(({ country, flag, name }) => (
-          <div className={styles.country} key={country}>
+          <div
+            className={styles.country}
+            key={country}
+            onClick={() => focusMarkersInCountry(country)}
+            onKeyDown={(e) => e.key === 'Enter' && focusMarkersInCountry(country)}
+            role="button"
+            tabIndex={0}
+          >
             {flag} {name}
           </div>
```

Comment:
This should be a button, or possibly an anchor

Reply:
Good call, I figured `role="button"` may be sufficient but using a button is definitely the better solution.